core/spec.md

```diff
@@ -4061,10 +4063,42 @@ and the following Resource level attributes:
 
 - Constraints:
   - REQUIRED.
-  - If present, it MUST be a case sensitive value from the model defined
+  - If present, it MUST be a case-sensitive value from the model-defined
     enumeration range.
   - When not specified, the default value MUST be `none`.
   - The enumeration range MUST include `none` as a valid value.
+  - The value MAY change only when the new `compatibility` value can be
+    enforced across all existing Versions. If that's not the case, the
+    server MUST generate an error.
+
+#### `compatibilityauthority` Attribute
+- Name: `compatibilityauthority`
+- Type: Boolean
+- Description: Indicates the authority that enforces the `compatibility`
+  value defined by the owning Resource.
+
+  This specification defines the following enumeration values. Implementations
+  MAY choose to extend this list.
+  - `external` - The compatibility is enforced by an external authority.
+  - `server` - The compatibility is enforced by the server.
+
+  When set to `server`:
+  1. The server MUST reject the request if the server cannot enforce the
```

Comment:
s/reject the/reject any/

otherwise I wasn't sure which request you were referring to.

Reply:
actually, which request were you thinking of that isn't covered by the next 2 bullets?

Reply:
I was thinking of a request to set the compatibilityauthority to server. I'll clarify this.